adapters/rainbowkit-connector/package.json

```diff
@@ -32,10 +32,9 @@
     "lint": "eslint src --ext .ts"
   },
   "dependencies": {
-    "@blocto/sdk": "^0.5.5-beta.2",
-    "@rainbow-me/rainbowkit": "^1.0.1",
-    "viem": "^0.3.50",
-    "wagmi": "1.1.0"
+    "@blocto/sdk": "^0.5.5-beta.3",
```

Comment:
since rainbowkit's bloctoConnector has been replaced with `@blocto/wagmi-connector`, perhaps `@blocto/sdk` is no long required for as dependiencies.